package.json

```diff
@@ -75,19 +75,34 @@
         },
         "commands": [
             {
-                "command": "autoHide.toggleHideSideBar",
+                "command": "autoHide.toggleHideSideBarWorkspace",
```

Comment:
Leave all setting and commands the same, other wise its a breaking change.

Reply:
Workspace commands now reverted back 👍